wger/core/views/user.py

```diff
@@ -520,12 +538,22 @@ def get_context_data(self, **kwargs):
         '''
         Pass other info to the template
         '''
+        
         context = super(UserListView, self).get_context_data(**kwargs)
         context['show_gym'] = True
         context['user_table'] = {'keys': [_('ID'),
                                           _('Username'),
                                           _('Name'),
                                           _('Last activity'),
-                                          _('Gym')],
+                                          _('Gym'),
+                                          _('Account Status'),
+                                          _('Change Status'),
+                                          _('Delete Account')],
                                  'users': context['object_list']['members']}
+        '''
```

Comment:
Use _pound_ comments here. Docstrings are appropriate for documenting functions/classes.